apps/src/StudioApp.js

```diff
@@ -1,5 +1,14 @@
-// Globals:
-//   Blockly
+/* jshint
+  strict: true,
+  curly: true,
+  funcscope: true,
+  newcap: true,
+  nonew: true,
+  shadow: false,
+  unused: true
+*/
+/* global Blockly: true */
```

Comment:
We do this for JSHint globally, but we could change to doing it locally (or use local overrides to help): Instead of just using a comment to document what globals are used here, we can whitelist them with JSHint and get complaints if we introduce new ones.